client/src/components/Review.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Review = (props) => {
+  const {
+    ID, STARS, REVIEW_DATE, REVIEW_TEXT, USEFUL, FUNNY, COOL, user,
+  } = props.review;
+  // const STARS = props.review.STARS;
+  // const REVIEW_DATE = props.review.REVIEW_DATE;
+  // const REVIEW_TEXT = props.review.REVIEW_TEXT;
+  // const USEFUL = props.review.USEFUL;
+  // const FUNNY = props.review.FUNNY;
+  // const COOL = props.review.COOL;
+  // const user = props.review.user;
+  let userPhotoIdUrl = './Pictures/default.png';
+  if (user.PHOTO_ID !== 'NULL') {
+    userPhotoIdUrl = ['./Pictures/', user.PHOTO_ID, '.png'].join('');
```

Comment:
Consider using a template literal here.
userPhotoIdUrl = `./Pictures/${user.PHOTO_ID}.png`;
Non-blocking.

Reply:
I remember changing it to this because the style guide didn't like using + to concatenate, but yes template literal is better.